pkg/cgroup/resolve_container.go

```diff
@@ -193,26 +242,27 @@ func getPathFromcGroupID(cgroupID uint64) (string, error) {
 		if !dentry.IsDir() {
 			return nil
 		}
-		getCgroupID, err := utils.GetCgroupIDFromPath(byteOrder, path)
+		getCgroupID, err := utils.GetCgroupIDFromPath(utils.DetermineHostByteOrder(), path)
```

Comment:
Why do we need to call DetermineHostByteOrder every call here?

Reply:
No. I will set this back

Reply:
I will update this